### PR TITLE
Add lower value for plastic strain at failure for /FAIL/JOHNSON

### PR DESCRIPTION
**Pull request by vdavaze**
#### Description of the feature or the bug

Add a lower value for plastic strain at failure for /FAIL/JOHNSON and fix a major memory issue in element buffer for beam failure (leading to segfault).

#### Description of the changes

Add EPSF_MIN parameter and fix the element buffer problem in elbuf_ini.

### CODE REVIEW

File: starter/source/elements/elbuf_init/elbuf_ini.F [hunk 1]
```diff
@@ -1993,8 +1994,8 @@ SUBROUTINE ELBUF_INI(ELBUF_TAB ,MAT_PARAM,
               NVARTMP = ELBUF_TAB(NG)%BUFLY(IL)%NVARTMP
               NUVAREOS= ELBUF_TAB(NG)%BUFLY(IL)%NVAR_EOS
               NPTT    = ELBUF_TAB(NG)%BUFLY(IL)%NPTT
-              NFAIL   = ELBUF_TAB(NG)%BUFLY(IL)%NFAIL
-              IF (NFAIL == 1) ALLOCATE(BUFLY%FAIL(NPTR,NPTS,NPTT))                
+              NFAIL   = ELBUF_TAB(NG)%BUFLY(IL)%NFAIL                                                
+              IRUPT   = MAT_PARAM(IMAT(IL))%FAIL(1)%IRUPT   ! modele de rupture
```
Comment (sebastienVilleneuve) [inline starter/source/elements/elbuf_init/elbuf_ini.F:1998]: can you translate to english :-) 

Thanks,
Sébastien

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
@vdavaze 

Thanks for the contribution , some minor change to perform , before merging.

Thanks,
Sébastien

File: starter/source/elements/elbuf_init/elbuf_ini.F (hunk 1 above)
Comment (vdavaze) [reply starter/source/elements/elbuf_init/elbuf_ini.F]: Hi @sebastienVilleneuve , 

Sorry for this, bad copy paste of a line containing french comment. Thanks a lot for the review !

Comment (sebastienVilleneuve) [reply starter/source/elements/elbuf_init/elbuf_ini.F]: Thanks @vdavaze

**Review by sebastienVilleneuve — APPROVED** [review]

**Pull request merged**